### ISSUE TITLE
Include IGA in the default build

### ISSUE THREAD

**Issue by dvrogozh** [intel/intel-graphics-compiler#36]
Please, include IGA into default build. This is required to build media-driver VP shaders/kernels: https://github.com/intel/media-driver/tree/master/media_driver/agnostic/gen11_icllp/vp/kernel/Source

**Comment by iwwu** [comment]
IGC already included IGA library (i.e. iga64.so) in the build. We are looking into including IGA executable (i.e. iga64.exe) as well.

**Comment by dvrogozh** [comment]
To build IGA manually:
```
cd visa/iga
mkdir build && cd build
cmake ..
make
```

**Comment by iwwu** [comment]
iga64 is stored under /usr/local/bin by default

**Comment by dvrogozh** [comment]
I was building IGC with (on CentOS 7):
```
scl enable devtoolset-4 llvm-toolset-7 -- cmake -DIGC_OPTION__OUTPUT_DIR=../igc-install/Release -DCMAKE_INSTALL_PREFIX=/home/dvrogozh/git/github/cmc ../igc
make -j8
make install
```

And I got iga64 installed twice:
```
-- Install: /home/dvrogozh/git/github/cmc/bin/iga64
-- Install: /home/dvrogozh/git/github/cmc/./iga64
```

The second one is unexpected. Can this be, please, fixed?

**Comment by iwwu** [comment]
Your request regarding this is already addressed in our pending patch. I will update this issue when it's fixed.
Install: /home/dvrogozh/git/github/cmc/./iga64

**Comment by dvrogozh** [comment]
Sounds good. Thank you.

**Comment by dvrogozh** [comment]
@iwwu : I was just rebuilding IGC and I believe the pending patch you were talking about has landed (I am on b9d9c185f08b883ce7a32d4bf49ebabbe3c7d4c8). Can you, please, confirm and specify commitid where fix presents?

**Comment by iwwu** [comment]
It's on 64632fa (tag: igc_release_2019-01-15)
